src/fides/api/models/privacy_notice.py

```diff
@@ -108,6 +108,36 @@ class PrivacyNoticeRegion(Enum):
     eu_fi = "eu_fi"  # finland
     eu_se = "eu_se"  # sweden
 
+    gb_eng = "gb_eng"  # england
+    gb_sct = "gb_sct"  # scotland
+    gb_wls = "gb_wls"  # wales
+    gb_nir = "gb_nir"  # northern ireland
+
+    is_1 = "is_1"  # iceland, capital region
+    is_2 = "is_2"  # iceland, southern peninsula
+    is_3 = "is_3"  # iceland, western region
+    is_4 = "is_4"  # iceland, westfjords
+    is_5 = "is_5"  # iceland, northwestern region
+    is_6 = "is_6"  # iceland, northeastern region
+    is_7 = "is_7"  # iceland, eastern region
+    is_8 = "is_8"  # iceland, southern region
+
+    no_42 = "no_42"  # norway, agder
+    no_34 = "no_34"  # norway, innlandet
+    no_15 = "no_15"  # norway, møre og romsdal
+    no_18 = "no_18"  # norway, nordland
+    no_03 = "no_03"  # norway, oslo
+    no_11 = "no_11"  # norway, rogaland
+    no_54 = "no_54"  # norway, troms og finnmark
+    no_50 = "no_50"  # norway, trøndelag
+    no_38 = "no_38"  # norway, vestfold og telemark
+    no_46 = "no_46"  # norway, vestland
+    no_30 = "no_30"  # norway, viken
+    no_22 = "no_22"  # norway, jan mayen
+    no_21 = "no_21"  # norway, svalbard
+
+    li = "li"  # liechtenstein
```

Comment:
interesting, but couldn't they have `"is"` or `"no"` as the enum _values_ while the enum _name_ would be something different, so as to avoid the conflict with the python keywords?

i'm guessing you thought through this a fair amount and it's not an easy adjustment. and i know we've got some time pressure, so not trying to make more work than needed. but if there's no real requirement to have regions within iceland or norway, it just feels a bit odd to do this, especially because i think as a consequence we'll have `PrivacyExperience` records created for each region?

i may not be understanding this correctly. just struck me as a bit surprising to have all of these regions if the only purpose is to avoid a python conflict, i feel like there has to be some other way around it... 🤷

Reply:
You're right I can do "iceland" or similar for now. 

I learned today too the frontend location service is actually by region, and we need to do a larger discussion around how we're going to sync up around regions but that's likely next sprint.

Your point about creating excessive PrivacyExperience records when we still don't know how this is going to settle out is definitely valid.

Reply:
ah right norway "no" also was problematic for some reason? Anyway, to not spend too much time here, I'm using three letter country codes for both of those for now, but it is likely all these regions will get an overhaul in a future sprint.